aip/general/0193.md

```diff
@@ -69,13 +69,25 @@ and the method **should** put partial failure information in the metadata
 message. The errors themselves **must** still be represented with a
 [`google.rpc.Status`][Status] object.
 
+### Permission Denied
+
+If the user does not have permission to access the parent resource, regardless
+of whether or not it exists, the service **must** error with
+`PERMISSION_DENIED` (HTTP 403). Permission **must** be checked prior to
+checking if the parent resource exists.
+
+If the user does have proper permission, but the requested parent resource does
+not exist, the service **must** error with `NOT_FOUND` (HTTP 404).
```

Comment:
@GregFurmanek @shwoodard I think the original text is a little confusing here, since before it was specifically in the context of not found. 

The guidance here would be better off as more general, I think:

- if the user does not have permissions on the resourced, return 403.
- otherwise return the appropriate error as indicated in this AIP.

WDYT?

Reply:
You have now moved the guidance into AIP-193 so it needs to apply for all Standard methods. In the case of Get and others, there is no parent. Please fix.

Reply:
done, PTAL.